packages/eas-build-job/src/artifacts.ts

```diff
@@ -0,0 +1,8 @@
+export enum ManagedArtifactType {
+  APPLICATION_ARCHIVE = 'APPLICATION_ARCHIVE',
+  BUILD_ARTIFACTS = 'BUILD_ARTIFACTS',
+  /**
+   * @deprecated
+   */
```

Comment:
Do you know why it is deprecated? I know it was `deperecated` before your PR but I'm wondering what the intention was here

Reply:
I think at some point we stopped uploading Xcode logs as "build artifacts" and started to treat them separately — not only do they land in a separate bucket (`turtle-v2-logs` vs `turtle-v2-artifacts`), but also they don't get renamed as other artifacts do (from `prefix` to `prefix/name.suffix`).